### DISCUSSION TITLE
Sample configs can't be found in Debian 10 backports package.

### DISCUSSION THREAD

**Discussion by PC-Admin** [moosefs/moosefs#402] [General]
```
$ sudo apt -t buster-backports install moosefs-master
$ ls /etc/mfs/
```

I'm not sure how to begin, all the instructions i find online say to look in /etc/mfs, yet there's nothing here. :P

**Comment by onlyjob** [comment]
Did you check the usual location `/usr/share/doc/moosefs-master/examples`?

Always check documentation in `/usr/share/doc/${package_name}`.

**Comment by PC-Admin** [comment]
Found them, thanks for the help. :)
